connect_node.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#USAGE: ./reconnect <project-name-as-in-hil> <node-name-as-in-hil> <iscsi-server-ip-address>
```

Comment:
@apoorvemohan I really feel we should set the iscsi-server-ip-address a variable here and set it in this file. It makes the user experience better when they don't have to type it

Reply:
I do not have any strong opinion against it. I am ok with either approaches.

Reply:
@mosayyebzadeh can you update this so that iscsi server is a variable that can be set in the file rather than taking it as a command line argument?